src/components/CustomInput/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { Text, TextInput, TouchableOpacity } from 'react-native';
+import { style } from '../../screens/AddNewItemScreen/style';
+import React, { FC, MutableRefObject } from 'react';
+
+export type CustomInputPropsType = {
+  onChangeText?: React.Dispatch<React.SetStateAction<string>>;
+  placeholder?: string;
+  value: string;
+  selectionColor: string;
+  title: string;
+  onPress: () => void;
+  editable: boolean;
+  inputRef?: MutableRefObject<TextInput | null>;
+};
+
+export const CustomInput: FC<CustomInputPropsType> = props => {
+  const {
+    onChangeText,
+    placeholder,
+    selectionColor,
+    value,
+    title,
+    onPress,
+    editable,
```

Comment:
remove editable prop 
disable input if you have onPress props

Reply:
fix it